### PR TITLE
Fetch Projects

### PR DESCRIPTION
**Pull request by wgranger**
This PR begins fetching projects from the TOVE backend. Currently, all projects are being returned from Tove as there are only a few in the database. Later, this will be changed to returning only those projects for the logged in user.

### CODE REVIEW

**Comment by shaunanoordin** [comment]
Storybook is currently broken for this branch; a fix has been implemented in #50 . This branch requires rebasing to the new master, or (in my case) I'll just monkey-patch the corrected config for testing.

File: src/screens/ProjectsIndex/ProjectsPageContainer.js [hunk 1]
```diff
@@ -12,30 +25,52 @@ export default function ProjectPageContainer () {
...
+        </Box>}
+
+      {store.projects.asyncState === ASYNC_STATES.LOADING &&
+        <Box justify='center' direction='row' gap='xsmall'>
+          <Text>Loading</Text>
+          <FontAwesomeIcon icon={faSpinner} spin />
+        </Box>}
+
+      {store.projects.asyncState === ASYNC_STATES.ERROR &&
+        <Text color='red' textAlign='center'>{error}</Text>}
```
Comment (shaunanoordin) [inline src/screens/ProjectsIndex/ProjectsPageContainer.js]: Status indicators: nice 👍

File: src/screens/ProjectsIndex/ProjectsPageContainer.js [hunk 2]
```diff
@@ -12,30 +25,52 @@ export default function ProjectPageContainer () {
...
+
+      {ownerProjects.length === 0 && collabProjects.length === 0 && store.projects.asyncState === ASYNC_STATES.READY &&
+        <Text textAlign='center'>
+          We couldn't find any transcription projects you participate in.
```
Comment (shaunanoordin) [inline src/screens/ProjectsIndex/ProjectsPageContainer.js]: Edge cases: niiice 👍

**Comment by wgranger** [comment]
Superagent vs. Frisbee

This was a suggestion from @srallen. I believe the idea was to try out Frisbee since it implements the [new fetch api](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API). The monorepo may also start using Frisbee in the future if we have a good experience here.

**Pull request merged**